.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: python
+python:
+  - "2.7"
+script: "python setup.py test"
```

Comment:
While you are here, why don't you add flake8 to the list of scripts to run? maybe you can put it before the tests run, so that if flake8 fails the tests won't get executed.